homework/shirley_homework_2.py

```diff
@@ -0,0 +1,147 @@
+import pandas as pd
+import numpy as np
+import psycopg2
+from sqlalchemy import create_engine
+
+
+#readng the raw csvs 
+raw_employees = pd.read_csv('/Users/shirleyxueyinghe/programming/barcelona_gse/databases/warehousing-operational-databases/homework/extracts/employees.csv')
```

Comment:
No absolute paths!

Reply:
And technically I did want you to EXTRACT from the operational database, not from these original files. That was supposed to be the E in ETL!